packages/kotti-ui/source/kotti-style/forms/_label.scss

```diff
@@ -14,22 +14,3 @@
 		line-height: var(--unit-4);
 	}
 }
-
-.form-label--compact {
```

Comment:
dead and deprecated since ages (ideally all form-input, form-group, form classes in kotti style should be gone in favor of the component)